.circleci/config.yml

```diff
@@ -415,11 +415,11 @@ install_multicluster_deps: &install_multicluster_deps
     name: "Install multicluster deps"
     command: |
       sed -i -e "s/172.18.0.2/$DEFAULT_DEX_IP/g;s/localhost/kubeapps-ci.kubeapps/g" ./docs/user/manifests/kubeapps-local-dev-dex-values.yaml
-      helm repo add stable https://charts.helm.sh/stable
+      helm repo add dex https://charts.dexidp.io
```

Comment:
After removing `stable`, we should also move `openldap`:
`error: failed to download "stable/openldap" (hint: running `helm repo update` may help)`

Reply:
Yeah, I didn't want to try to change ldap. Just updated and fired off yesterday before moving onto something else, and forgot that we'll need stable still. It'll still fail after updating I expect as there's a new UI step in dex when logging in (giving consent to email being passed to 3rd party).